keras/ops/operation.py

```diff
@@ -280,3 +280,11 @@ def _post_build(self):
     def _setattr_hook(self, name, value):
         """Can be overridden for per backend post build actions."""
         return name, value
+
+    def _post_track_variable(self, variable):
+        """Can be overridden for per backend post track actions."""
+        pass
+
+    def _post_untrack_variable(self, variable):
```

Comment:
Two hooks have been introduced to enable the postprocessing of `_track_variable` and `_untrack_variable` in torch backend